src/app/pages/PhotoGalleryPage/index.test.jsx

```diff
@@ -7,13 +7,19 @@ import assocPath from 'ramda/src/assocPath';
 import { ServiceContextProvider } from '#contexts/ServiceContext';
 import { RequestContextProvider } from '#contexts/RequestContext';
 import { ToggleContext } from '#contexts/ToggleContext';
-import { PhotoGalleryPage } from '..';
+import { PhotoGalleryPage as AsyncPhotoGalleryPage } from '..';
 import noOnwardJourneys from '#data/pidgin/cpsAssets/sport-23252855';
 import someCpsOnwardJourneys from '#data/azeri/cpsAssets/azerbaijan-44208474.json';
 import allCpsOnwardJourneys from '#data/pidgin/cpsAssets/tori-49221071.json';
 import pglAboutData from '#data/afaanoromoo/cpsAssets/oduu-41217768';
 import getInitialData from '#app/routes/cpsAsset/getInitialData';
 
+let PhotoGalleryPage;
+
+beforeAll(async () => {
```

Comment:
Is this only required for the photo gallery pages?

Reply:
I've removed this for now, the approach we are using to fix the tests should be on this PR https://github.com/bbc/simorgh/pull/7344 , having it in this one was a mistake.